hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java

```diff
@@ -379,6 +379,15 @@ private ServicesApp internalInit(
             app.initializationFlow().runWith(this, bootstrapProps);
             if (trigger == RESTART && isUpgrade) {
                 app.stakeStartupHelper().doUpgradeHousekeeping(networkCtx(), accounts(), stakingInfo());
+                // FIXME: uncomment after migration to 0.37+
```

Comment:
All TODOs and FIXMEs should be filed as github tickets, with links put to code comments.